### PR TITLE
Auto patcher

### PR DESCRIPTION
**Pull request by jakobrosenberg**
This PR should remove any dependencies for NPM and node build tools on the host machine.

- Added [custom implementation](https://github.com/pycom/pymakr-vsc/pull/158/commits/fa7be26ac67d0dedd4c18ac1e00a11662d0a3656) of [prebuild-install](https://www.npmjs.com/package/prebuild-install)
- Removed dependencies for build tools on host
- Replaced unit test with integration tests which run in an instance VSCode (ensures Electron version is correct)
- Cleaned up and simplified the package further
  - cleaned up conflicting specification standards
  - removed unnecessary babel build steps
  - removed bindings builder and added auto downloader https://github.com/pycom/pymakr-vsc/commit/fa7be26ac67d0dedd4c18ac1e00a11662d0a3656

### CODE REVIEW

File: src/main/terminal.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 'use babel';
 var vscode = require('vscode');
-import Logger from '../helpers/logger.js'
-import ApiWrapper from '../main/api-wrapper.js';
+const Logger = require('../helpers/logger.js')const Logger = require('../helpers/logger.js')
```
Comment (elmasria) [inline src/main/terminal.js:3]: I think this is an issue?

File: src/main/panel-view.js [hunk 2]
```diff
@@ -3,13 +3,13 @@
 var vscode = require('vscode');
 import { StatusBarAlignment, window } from 'vscode';
 
-import Term from './terminal';
-import ApiWrapper from '../main/api-wrapper.js';
+const Term = require('./terminal');const Term = require('./terminal')const Term = require('./terminal')
+const ApiWrapper = require('../main/api-wrapper.js');const ApiWrapper = require('../main/api-wrapper.js')
```
Comment (elmasria) [inline src/main/panel-view.js:7]: Check this one!

File: src/connections/pyserial.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
 'use babel';
```
Comment (elmasria) [inline src/connections/pyserial.js:1]: maybe we have to remove this at this level!

File: src/config.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 'use babel';
```
Comment (elmasria) [inline src/config.js:1]: this one too!

File: src/board/sync.js [hunk 5]
```diff
@@ -1,15 +1,15 @@
 'use babel';
 
-import Shell from './shell.js'
-import Config from '../config.js'
+const Shell = require('./shell.js')const Shell = require('./shell.js')const Shell = require('./shell.js')const Shell = require('./shell.js')const Shell = require('./shell.js')const Shell = require('./shell.js')
+const Config = require('../config.js')const Config = require('../config.js')const Config = require('../config.js')const Config = require('../config.js')const Config = require('../config.js')
```
Comment (elmasria) [inline src/board/sync.js:4]: check this

**Review by elmasria — COMMENTED** [review]
I see a lot of duplication please check them. Thank you

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -1,108 +1,108 @@
...
-#        run: |
-#          npm ci
-#          npm prune --production
-#
-#      - name: Package Extension
-#        run: npx vsce package
-#
-#      - name: Publish Extension
-#        run: npx vsce publish --pat ${{ secrets.VSCE_TOKEN }}
+# name: Build Pymakr
```
[the remark this answers is not in the document]
Comment (jakobrosenberg) [reply .github/workflows/main.yml:1]: Will do. Only kept it as a reference for the tests and possible automated publish

File: src/main/terminal.js (hunk 1 above)
Comment (jakobrosenberg) [reply src/main/terminal.js:3]: Yes. That look like trouble. I'll get it fixed.

File: src/main/panel-view.js (hunk 2 above)
Comment (jakobrosenberg) [reply src/main/panel-view.js:7]: Thanks for catching this. The search/replace bugged up and made a bunch of these double statements. I thought I had found them all. I'll clean them up.

**Comment by jakobrosenberg** [comment]
For some reason the `src` folder was still present on github even though I had deleted it locally. Had to clone a new repo and delete it again. This should clear all `src` related PR comments.

I'll do a search and replace for all 'use babel'; entries.

**Review by elmasria — APPROVED** [review]
Thank you

**Pull request merged**